pallets/liquidate/src/lib.rs

```diff
@@ -291,33 +295,31 @@ pub mod module {
                     // if (collateral_total_value * liquidation_threshold)/(debt_total_value) < 1 ,execute liquidation
                     let mut processing = true;
                     let collateral_liquidation_threshold_value = classify_collaterals.iter().fold(
-                        Price::MIN,
+                        MIN_PRICE,
                         |acc,&(_,_,_,total_sum_price,liquidation_threshold)|
 							// acc + total_sum_price * liquidation_threshold
-							match total_sum_price
-								.checked_mul(liquidation_threshold)
-								.and_then(|r| r.checked_div(RATE_DECIMAL))
-								.and_then(|r| r.checked_add(acc))
-								.ok_or(Error::<T>::CaculateError)
-							{
-								Ok(v) => v,
-								Err(e) => {
-									log::error!("error calculate liquidation threshold: {:?}",e);
-									processing = false;
-									acc
-								}
+								match total_sum_price.saturating_mul(liquidation_threshold.into())
```

Comment:
I think `total_sum_price` is u128, I'm confusing when convert `liquidation_threshold.into()` into u128, if the value will change? 
because we suppose `liquidation_threshold` equals 80% but after an `into()` method, what the actual value is?

Reply:
total_sum_price is FixedU128, liquidation_threshold is Permill. 
FixedU128::saturating_from_integer(100).saturating_mul(Permill::from_percent(50).into()) 
equal to FixedU128::saturating_from_integer(50).

Reply:
ok, get it